packages/fuel-indexer-schema/src/lib.rs

```diff
@@ -260,9 +262,9 @@ impl FtColumn {
                 None => String::from(NULL_VALUE),
             },
             FtColumn::Blob(value) => match value {
-                Some(val) => {
+                Some(fuel_indexer_types::Blob(val)) => {
```

Comment:
Using `impl AsRef<[u8]> for Blob` prevents needing to make this change.

Reply:
Ok let me try this!